governance.md

```diff
@@ -96,6 +98,13 @@ Duties:
 * Address questions or concerns from the community about project culture, structure, and clarifications of roles, responsibilities and duties
 * Ratify new SIGs
 
+Qualifications:
+
+* Nominated and Appointed by Steering Committee
```

Comment:
See comments above. If an SC member is up for re-election, do they get to take part in the nomination and/or vote?

Reply:
I guess we should add language here - my thought is that a member cannot vote for their own re-election.

Reply:
Does that create the potential for a split vote? (chair person breaks a tie?)

Reply:
> Does that create the potential for a split vote?

Er, um, indeed. Hrm. Chairperson breaking the tie sounds good to me

Reply:
I think I misread the initial question. If an SC member is up for re-election, they have full voting rights until they are removed. That means we need to prepare for elections before the end of the year. The person up for re-election shouldn't be able to vote for themselves though. And then in that case, the chairperson breaks the tie. Thoughts?

Reply:
> The person up for re-election shouldn't be able to vote for themselves though. And then in that case, the chairperson breaks the tie. Thoughts?

That was my reading of @ajordens' suggestion - +1 to that from me

Reply:
What happens if the chairperson is the one who is up for reelection and there is a tie?  I would propose instead that we simply state that a majority of the SC is needed to be elected (or reelected), which means that a person is not elected in the case of a tie.  If the committee is that split about a candidate, then I worry about their being a cohesive part of the leadership.

Reply:
@aglover I don't see this thread reflected in the changes to this file.  Is that intentional?